source/extensions/filters/http/cache/http_cache.h

```diff
@@ -256,6 +256,7 @@ class HttpCache {
 
   virtual ~HttpCache() = default;
 };
+using HttpCachePtr = std::shared_ptr<HttpCache>;
```

Comment:
Guess I'll say it again since there's no way to unmark a conversation as resolved:
Envoy style has been moving towards naming this HttpCacheSharedPtr to avoid confusion as to whether it's shared/unique/bare at all the use-sites.